umpost/um2netcdf.py

```diff
@@ -427,6 +433,65 @@ def process(infile, outfile, args):
             cubewrite(c, sman, args.compression, args.use64bit, args.verbose)
 
 
+def get_grid_type(ff):
+    """
+    Returns grid type from a fields file.
+
+    Parameters
+    ----------
+    ff : an open fields file.
+
+    Returns
+    -------
+    String code for grid type, or raise a UMError.
+    """
+    staggering = ff.fixed_length_header.grid_staggering
+
+    if staggering == 6:
+        return GRID_END_GAME
+    elif staggering == 3:
+        return GRID_NEW_DYNAMICS
+    else:
+        raise UMError(f"Unable to determine grid staggering from header '{staggering}'")
+
+
+def get_grid_spacing(ff):
+    """
+    Helper function for accessing grid spacing variables.
+
+    Parameters
+    ----------
+    ff : an open fields file.
+
+    Returns
+    -------
+    (row_spacing, column spacing tuple)
```

Comment:
```suggestion
    (row_spacing, column_spacing) tuple
```
Some type information could useful but it is also not super necessary.

Reply:
Fixed the comment & added datatypes to description.